client/app/dedicated/server/dashboard/dedicated-server-dashboard.routes.js

```diff
@@ -0,0 +1,13 @@
+import template from './dedicated-server-dashboard.html';
+
+angular.module('App').config(($stateProvider) => {
```

Comment:
We encourage you to add explicit ngInject wherever you can.

```suggestion
angular.module('App').config(/* @ngInject */($stateProvider) => {
```

Reply:
Hmm I forgot these ones...

Reply:
It's done